README.md

```diff
@@ -443,6 +443,7 @@
       return `How are you, ${name}?`;
     }
     ```
+  - [6.5](#6.5) <a name='6.5'></a> Never use eval() on a string, it opens too many vulnerabilities.
```

Comment:
Let's also mention the `Function` constructor, which while safer than `eval`, is still `eval` and therefore evil